Modules/socketmodule.c

```diff
@@ -624,6 +618,60 @@ find_module_state_by_def(PyTypeObject *type)
     return get_module_state(mod);
 }
 
+#define UNSIGNED_INT_CONVERTER(NAME, TYPE)                          \
+int                                                                 \
```

Comment:
`static int` would be better here, no?

```suggestion
static int                                                          \
```